keras/utils/vis_utils.py

```diff
@@ -280,12 +280,11 @@ def format_shape(shape):
         inputlabels = format_shape(layer.input_shape)
       elif hasattr(layer, 'input_shapes'):
         inputlabels = ', '.join(
-            [format_shape(ishape) for ishape in layer.input_shapes])
+          [format_shape(ishape) for ishape in layer.input_shapes])
       else:
         inputlabels = '?'
-      label = '%s\n|{input:|output:}|{{%s}|{%s}}' % (label,
-                                                     inputlabels,
-                                                     outputlabels)
+      label = '%s|{input:|output:}|{{%s}|{%s}}' % (
```

Comment:
Likewise indent should be 4. You can shorten the line by using a f-string instead of `%`